### DISCUSSION TITLE
Multiple accounts for the same service

### DISCUSSION THREAD

**Discussion by birnbeidla** [avantrec/soco-cli#29] [Q&A] [answered]
Hi!

I'm using a Spotify family account and have all members added to my Sonos system.
Is it possible with soco-cli to select the account that is used to play a song?

**Comment by pwt** [comment, marked as answer]
That's a good question.

Right now, the only support for music services (e.g., Spotify) is via Sonos favourites (for single tracks only), Sonos Playlists, and Sharelinks.

As none of these options are music service account-specific, but instead apply to the Sonos system as a whole, SoCo-CLI doesn't do anything to specify which music service account to use when playing items, so there's no current way to specify which music service account to use.

I actually don't know how it behaves when there are multiple accounts for a given music service, so I'll have to do some testing. Does SoCo-CLI playback of Spotify content work OK for you with multiple accounts in your system? E.g.: does this work to queue a track (Everything Now, by Arcade Fire, in case you're wondering):

`sonos <speaker_name> sharelink "https://open.spotify.com/track/07Ugv4TA7PvQfoqetnmLrS?si=24760efc87ad4398"`

Currently, there's no general support for music services, e.g., searching, accessing music service playlists directly, playing back content directly without having to go via Sonos Playlists or Sharelinks, etc. Hopefully this will improve in the next few months due to work underway in the underlying SoCo library. As part of this, there'll be a music service authentication step which will bind the use of each music service to a specific account.
